controller_report_xls/controllers/main.py

```diff
@@ -38,6 +38,96 @@
 _logger = logging.getLogger(__name__)
 
 
+def is_float(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_integer(value):
+    try:
+        int(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_formatted_number(value):
+    set_sign = set([',', '.', '-'])
```

Comment:
Please check here, this is the first approach,

that is considered here

https://github.com/vauxoo-dev/addons-vauxoo/blob/236bb8277113a9548df25d6097915c533658b4cd/controller_report_xls/controllers/main.py#L109-L118